Sources/web3swift/Web3/Web3+Methods.swift

```diff
@@ -64,6 +71,22 @@ public enum JSONRPCmethod: String, Encodable {
                 return 0
             case .getTxPoolInspect:
                 return 0
+            case .newPendingTransactionFilter:
+                return 0
+            case .getFilterChanges:
+                return 1
+            case .newFilter:
+                return nil
+            case .newBlockFilter:
+                return 0
+            case .getFilterLogs:
+                return nil
```

Comment:
What is the difference between `nil` and `0`? 
Looks like the meaning is the same but now `requiredNumOfParameters` has to be unwrapped.

Reply:
`nil` means that there are variable number of parameters

Reply:
Could you please add a documentation comment to `requiredNumOfParameters` mentioning that explanation?

Reply:
Added comment in [54cf7e9](https://github.com/skywinder/web3swift/commit/54cf7e9920456426c67612fbcb2e4233145c60a2)